### ISSUE TITLE
LEED Tables service water heating doesn't appear

### ISSUE THREAD

**Issue by JasonGlazer** [NREL/EnergyPlus#5613]
Tables EAp2-4/5, EAp2-17a, EAp2-17b

Service Water Heating doesn't show up as an end use in these tables unless a WaterHeater object is providing SHW. If another type of equipment such as a BoilerHotWater or HeatExchangerFluidToFluid is providing SHW to the WaterUseConnection and WaterUseEquipment the energy use will be zero.
- [ ] Defect file added (list location of defect file here)
- [ ] Ticket added to Pivotal for defect (development team task)
- [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by MatthewSteen** [comment]
**This also affects the End Uses table.** The energy use of boilers used for SHW will end up in the Heating end use instead of Water Systems.

**Comment by JasonGlazer** [comment]
@MatthewSteen or @mjwitte  I don't see any example files that are configured this way. Do you have any files that use BoilerHotWater and HeatExchangerFluidToFluid to provide SHW that you can provide as defect files?

**Comment by mjwitte** [comment]
5ZoneWaterSystems has a boiler serving a water heater.

**Comment by JasonGlazer** [comment]
Any without a water heater?

**Comment by mjwitte** [comment]
No, can't think of any without a waterheater object.  You could take WaterHeaterDHWPlantLoop and replace the waterheater:mixed with a boiler.

**Comment by JasonGlazer** [comment]
This issue is going to require allocating boiler energy use between two or more end uses if the demands on the hot water loop come from multiple end uses. It will be tricky to perform that allocation.

**Comment by MatthewSteen** [comment]
The reverse is also true. If a water heater is used for heating, which is a customary workaround for modeling air-to-water heat pumps in EnergyPlus, the energy use is reported in Water Systems rather than Heating. 

It's also worth pointing out that this issue isn't isolated to the LEED Summary report, but many of the other tabular reports (End Uses, Energy Meters, etc.).
